packages/@o3r/workspace/schematics/sdk/index.ts

```diff
@@ -48,6 +48,10 @@ function generateSdkFn(options: NgGenerateSdkSchema): Rule {
 
     const packageManager = getPackageManager({ workspaceConfig });
     const specExtension = options.specPackagePath ? path.extname(options.specPackagePath) : '.yaml';
+    // If spec path is relative to process.cwd, we need to make it relative to the project root
+    if (options.specPath?.startsWith('.')) {
```

Comment:
not covering all the cases, why not using `options.specPath && !path.isAbsolute(options.specPath)` ?

Reply:
I didn't use `path.isAbsolute` because `options.specPath` can also be an URL, but indeed `startsWith('.')` is not enough